### PR TITLE
[@types/react]  Add Valign attributes

### LINKED ISSUE #37435: [@types/react] valign missing on `<td>`
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @....

If you do not mention the authors the issue will be ignored.
`td: React.DetailedHTMLProps<React.TdHTMLAttributes<HTMLTableDataCellElement>, HTMLTableDataCellElement>;` at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2603 is missing valign as per [the spec](https://www.w3schools.com/tags/att_td_valign.asp)

### PR DESCRIPTION
**Pull request by jakesylvestre**
Fixes #37435 by adding the [valign](https://www.w3schools.com/tags/att_td_valign.asp) attribute to `@types/react`

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by jakesyl** [comment]
Failing travis test appears to not be caused by this PR.

**Comment by jakesyl** [comment]
image

**Comment by jakesyl** [comment]
image

By the way, I ran the tests locally and they passed

**Review by ferdaber — APPROVED** [review]
I would ping a maintainer to get it manually merged if Travis erroring out.

**Comment by jakesyl** [comment]
Got it @ferdaber and thanks for the approval. Guessing you mean the dynamically typed maintainers.

Ping:
@sandersn 

If I don't hear back for a few days I can always repush and have travis rerun.

Thanks!

**Comment by andrewbranch** [comment]
@sandersn pushed a fix for this intermittent npm error yesterday. I just requeued Travis.

**Comment by jakesyl** [comment]
Awesome, thanks @andrewbranch!

**Pull request merged**
